packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,111 @@
+---
+id: Release highlights
+title: Release highlights
+section: get-started
+---
+
+import './get-started.css';
+
+# 2023.06
+[View full details for this release on GitHub](https://github.com/patternfly/patternfly-react/releases).
+
+## Highlights 
+
+To upgrade your codebase to align with this release, [refer to our upgrade guide](/get-started/upgrade).
```

Comment:
This feels misplaced here as part of highlights. Maybe a separate heading about "Upgrading from PatternFly 4?"

Reply:
I moved it into a very brief "intro" sentence, let me know what you think